### PR TITLE
Hid tooltip if no subscores

### PR DESCRIPTION
**Pull request by lindaicing**
Hid tooltip when no subscores available,
Fixed some code formatting

Hovered on a page with subscores:
Screen Shot 2020-07-21 at 11 48 24 PM

Hovered on a page with no subscores:
Screen Shot 2020-07-21 at 11 48 55 PM

### CODE REVIEW

File: Extension/popup.js [hunk 1]
```diff
@@ -10,35 +12,38 @@ function loadExtension() {
...
+                hasSubscore = true;
             }
             var newHeight = "height:"+fullheight+"px;";
             document.body.style = newHeight;
         }
 
         //Change sitename
         document.getElementById("siteurl").innerHTML = response.ethicliStats.name;
-        if(response.ethicliStats.name == null){
+        if(response.ethicliStats.name == null) {
```
Comment (CubeOfCheese) [inline Extension/popup.js:46]: Missing space

File: Extension/popup.js [hunk 2]
```diff
@@ -97,9 +102,14 @@ window.onload = function() {
         somethingWrong();
     });
 
+    
     document.getElementById("scores").onmouseover = function() {
-        document.getElementById("subscoreTip").style.left = (event.clientX-30)+"px";
-        document.getElementById("subscoreTip").style.top = (event.clientY-30)+"px";
+        if(hasSubscore){
```
[the remark this answers is not in the document]
Comment (CubeOfCheese) [reply Extension/popup.js:107]: This line is still not consistent with the rest

File: Extension/popup.js [hunk 3]
```diff
@@ -10,35 +12,38 @@ function loadExtension() {
...
-            if(response.ethicliStats.animalsScore == 0.0){
+            if (response.ethicliStats.animalsScore == 0.0) {
                 fullheight= fullheight-50;
                 document.getElementById("animalSection").style="display:none;";
             }
-            if(response.ethicliStats.environmentScore == 0.0 &&
+            if (response.ethicliStats.environmentScore == 0.0 &&
                 response.ethicliStats.laborScore == 0.0 &&
                 response.ethicliStats.animalsScore == 0.0
             ){
```
Comment (CubeOfCheese) [inline Extension/popup.js:32]: Add space here

**Comment by lindaicing** [comment]
> Sorry for being nitpicky about the style but it makes it more readable if style is consistent

No worries at all! Keeps us more organized

**Pull request merged**